### ISSUE TITLE
Reusable Version Info Fails Silently In Docker Build for GHCR

### ISSUE THREAD

**Issue by cmarshak** [ASFHyP3/actions#202]
Related to this action: https://github.com/ASFHyP3/actions?tab=readme-ov-file#reusable-docker-ghcryml

Occurs when `setuptools_scm` not installed and version info fails silently. Causes breaking changes that are hard to parse later.

![image](https://github.com/user-attachments/assets/bf22a078-9bcf-48c9-aeae-23b5f7a920cc)

**Comment by jhkennedy** [comment]
Bah, I just wasted a bunch of time because of this -- definitely need to fix.
